config/samples/dspa_simple_v2.yaml

```diff
@@ -4,10 +4,32 @@ metadata:
   name: sample
 spec:
   dspVersion: v2
+  apiServer:
+    deploy: true
+    image: gcr.io/ml-pipeline/api-server:2.0.2
+  persistenceAgent:
+    deploy: true
+    image: gcr.io/ml-pipeline/persistenceagent:2.0.2
+  scheduledWorkflow:
+    deploy: true
+    image: gcr.io/ml-pipeline/scheduledworkflow:2.0.2
+  mlmd:
+    deploy: true
+    grpc:
+      image: gcr.io/tfx-oss-public/ml_metadata_store_server:1.14.0
+    envoy:
+      image: gcr.io/ml-pipeline/metadata-envoy:2.0.2
+  database:
+    disableHealthCheck: true
+    mariaDB:
+      deploy: true
   objectStorage:
+    disableHealthCheck: true
```

Comment:
Why disable by default?

Reply:
+1 do not disable by default

Reply:
I don't remember why. I probably forgot to enable it again.
Restored.